web/src/components/MalteHelpForm.tsx

```diff
@@ -102,13 +100,11 @@ const MalteHelpForm = ({ pageTitle, languageCode, cityCode, malteHelpFormOffer }
     }
   }
 
-  if (sendingStatus === 'successful') {
-    return (
-      <Alert severity='success' role='alert' action={<Link to={dashboardRoute}>{t('error:goTo.categories')}</Link>}>
-        {t('submitSuccessful')}
-      </Alert>
-    )
-  }
+  useEffect(() => {
+    if (sendingStatus === 'successful' || sendingStatus === 'failed' || sendingStatus === 'invalidEmail') {
+      setSnackbarOpen(true)
+    }
+  }, [sendingStatus])
```

Comment:
💭 Do we need to create a custom hook for this useEffect and the snackbarOpen state?

Reply:
i dunno, so far I used it only in two components 🤔

Reply:
See comment above, please avoid using `useEffect` for this. Side effects like this make the code harder to read and understand since its not clear when what happens.